Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    bali_view_components (0.14.0)
+    bali_view_components (0.14.1)
```

Comment:
Don't forget to update this version,

Reply:
I updated the version already 🤔

Reply:
you need to merge main first and then update. Main is on 0.15.0 already

Reply:
understood!